### DISCUSSION TITLE
How to solve the MachineAlreadyExistsError?

### DISCUSSION THREAD

**Discussion by Lix01786** [Kidel/Kathara#249] [Q&A] [answered]
Hello, after i downloaded the kathara-lab_small-internet-with-dns-and-web-server.zip i got this error message when i tried to run kathara lstart.
![image](https://github.com/KatharaFramework/Kathara/assets/150289836/4e1a22c8-cd58-4618-9f37-1c861acee059)

How can i fix this?

Thanks

**Comment by tcaiazzi** [comment, marked as answer]
Hi @Lix01786, 

Thanks for reaching out and sorry for the delay! 

The error is saying that you have already run a device called `as1r1` inside the network scenario. 

To re-run the network scenario, you only need to clean and restart it. 

To clean the scenario, you can use the `lclean` command:
```
kathara lclean
```

You can also clean & start the scenario at once using the `lrestart` command:
```
kathara lrestart
```

If you have any further problems, don't hesitate to ask!

Cheers, 
Tommaso

**Reply by Lix01786** [reply]
Now it's working, 
thanks for the help!
